### ISSUE TITLE
Implement Zebra and inboard Ge detector

### ISSUE THREAD

**Issue by helfrij** [acquaman/acquaman#2046]
Gameplan:
Implement the Zebra and the inboard detector for BioXASMain. Will not add the second, outboard detector yet as there are still being tests run to identify the source of intermittent noise. So, don't have to worry about differentiating between the inboard and the outboard detectors on Main, can use existing BioXASBeamline interface for now (this problem is being dealt with in #2035). The fast shutter is not yet installed on Main, so the aspects of Zebra setup specific to the shutter done in #1780 will be ignored here.
- [x] Initial Zebra setup
- [x] Double check that the Zebra software outputs correspond to actual connections -- ask David B.
- [x] Add instance of Zebra and Zebra trigger source to BioXASMainBeamline
- [x] Move Side specific settings from BioXAS32ElementGeDetector to new Side class
- [x] Double check Main inboard detector PV base name (PDTR1607-7-I21-02)
- [x] Create BioXAS32ElementGeDetector subclass for Main inboard detector, add to Main beamline
- [ ] Testing

**Comment by helfrij** [comment]
Waiting on confirmation of some PV names, and a quick check that the Zebra is set up correctly on Main. There may be some changes to the PV naming conventions for all detectors on BioXAS.

Time Estimate: 2 hours.

**Comment by helfrij** [comment]
The inboard detector PV base name is now available:
PDTR1607-7-I21-02

**Comment by dretrex** [comment]
PDTR?  For the Ge detector? o_O

**Comment by helfrij** [comment]
It stands for 'photon detector', I guess. The Ge detector PV names are getting revised on Side and Main to more closely resemble/match the drawings.

**Comment by helfrij** [comment]
Updated the PVs. The detector is connected, but doesn't initialize properly. I suspect there is a problem with the way I've set up the Zebra, waiting on help from David B.

**Comment by helfrij** [comment]
David B. needs to make some modifications to the Xspress3 software. Waiting on these changes before resuming testing.

**Comment by helfrij** [comment]
Gosia and I made some adjustments today to the Zebra device itself. 

We noticed that some of the output connections on the device didn't match the copy of the drawing I have, the cables were connected to Zebra inputs instead of outputs. Swapped the cables to the appropriate outputs. The detector then acquired in AM from the detector view, but it only appeared to dwell for about .7 seconds (didn't matter what the dwell time was actually set to).

We noticed that the cable plugged into TTL Out 1 went to the inboard detector electronics, and TTL Out 2 to the outboard detector electronics. The signal going to TTL Out 1 came from soft input 1, TTL Out 2 from Pulse 3. It looked like the pulse 3 block was triggering the outboard detector, instead of the inboard, and the inboard detector was getting its trigger from soft input 1, causing the above behavior. Swapping the TTL Out 1 and 2 cables seems to resolve the problem, the inboard detector is now operational in Acquaman. 

Need to talk to David B. about the connection changes. Neither of us (Gosia, me) have a good idea of how the outboard detector will be incorporated. Either the drawings need to be updated with the changes we made or (much more likely) the Zebra will need to be configured differently than currently set up.
